content/articles/domains-au.markdown

```diff
@@ -28,13 +28,15 @@ To check your priority category, or to see if there are other registrants eligib
 
 If there is no existing priority claim, meaning no matching registrations in any third-level .AU domain, you can register your domain right away, as with [any other domain](/articles/registering-domain).
 
-## Claiming a pre-existing domain
+### Claiming a pre-existing domain
 
-If you have an existing third-level .AU domain (.COM.AU, ORG.AU, etc.) with us, and you want to register the corresponding .AU name, we're planning to add support for this shortly after the start of the priority registration process.
+If you have an existing third-level .AU domain (.COM.AU, ORG.AU, etc.) that's _not_ registered with us, registrations are [now available](/articles/registering-domain). You will need to obtain a [Priority Token](https://priority.auda.org.au/) from auDA.
```

Comment:
Technically correct, we can also invite them to transfer the domain name to us to use the flow yet to be implemented :)

Reply:
I'm hesitant to suggest something for functionality that'a not yet available in a support article.

Reply:
Understood 👍  So we update as we go 🥳